examples/pytorch/pointcloud/pointnet/basic.py

```diff
@@ -0,0 +1,344 @@
+import torch
+import torch.nn.functional as F
+from torch.nn import Sequential as Seq, Linear as Lin, ReLU, Hardtanh, BatchNorm1d as BN
+from torch.nn.modules.utils import _single
+from torch.autograd import Function
+from torch.nn import Parameter
+import math
+from torch_geometric.data import DataLoader
+from torch_geometric.nn import PointConv, fps, radius, global_max_pool
+import numpy as np
+
+
+activations = {
+    'ReLU': ReLU,
+    'Hardtanh': Hardtanh
+}
+
+
+class MeanShift(torch.nn.Module):
```

Comment:
Most modules like this one in `basic.py` are never used. Please double-check and remove those unnecessary ones for simplicity.